### ISSUE TITLE
Attribute 'url' is nil when fetching Customer

### ISSUE THREAD

**Issue by ehannes** [my-codeworks/fortnox-api#32]
When requesting all `Customer`s from Fortnox through the gem, this is the first entry:

```
{"url":null,"address1":"","address2":"","city":"","country":null,"comments":null,"currency":null,"cost_center":null,"country_code":null,"customer_number":"1","delivery_address1":null,"delivery_address2":null,"delivery_city":null,"delivery_country":null,"delivery_country_code":null,"delivery_fax":null,"delivery_name":null,"delivery_phone1":null,"delivery_phone2":null,"delivery_zip_code":null,"email":"","email_invoice":null,"email_invoice_bcc":null,"email_invoice_cc":null,"email_offer":null,"email_offer_bcc":null,"email_offer_cc":null,"email_order":null,"email_order_bcc":null,"email_order_cc":null,"fax":null,"invoice_administration_fee":null,"invoice_discount":null,"invoice_freight":null,"invoice_remark":null,"name":"Test","organisation_number":"","our_reference":null,"phone1":null,"phone2":null,"price_list":null,"project":null,"sales_account":null,"show_price_vat_included":null,"terms_of_delivery":null,"terms_of_payment":null,"type":null,"vat_number":null,"vat_type":null,"visiting_address":null,"visiting_city":null,"visiting_country":null,"visiting_zip_code":null,"way_of_delivery":null,"your_reference":null,"zip_code":""}
```

When I do the same request with `curl`, this is the first entry:

```
{"@url":"https:\/\/api.fortnox.se\/3\/customers\/1","Address1":"","Address2":"","City":"","CustomerNumber":"1","Email":"","Name":"Test","OrganisationNumber":"","Phone":"","ZipCode":""}
```

Many attributes are filtered out when doing an `all`-request to Fortnox. Our gem sets these attributes to `null` instead, which can be incorrect but I think #25 is meant to fix that.

Instead, this issue concerns the `url` attribute. Gem sets it to `null`, but Fortnox returns `https:\/\/api.fortnox.se\/3\/customers\/1`. This is obviously wrong, but should our models even have a `url` attribute? That attribute declares where the resource can be found at Fortnox, but when using this gem we are not interested in querying directly against Fortnox. Or should we have some find method that takes the `url` as an argument and fetches that resource?

**Comment by ehannes** [comment]
See #50.

**Comment by ehannes** [comment]
Above mentioned issue is not the only problem. The other one is that the attribute is not called `url`, but `@url`.

**Comment by d-Pixie** [comment]
Fixed in #159
